index.js

```diff
@@ -68,7 +76,35 @@ function toggleColor(element) {
   }
 }
 
+
+
 /***** Deliverable 2 *****/
 
+const likeButton = document.querySelector("button")
+
+likeButton.addEventListener("click", function addLikes(event) {
+  const theLikes = document.querySelector(".likes") 
+  const addLikes = parseInt(theLikes.textContent) + 1
+  theLikes.textContent = `${addLikes} Likes`
```

Comment:
Looks good! The key here is that you have to *read* some information from the DOM to get the current number of likes, increment it, and then *write* it back to the DOM. Nice work!